### PR TITLE
Add Storj integration guide to docs

### PR DESCRIPTION
**Pull request by amozoss**
Doc change for more details on how to setup Storj with Dataverse

### CODE REVIEW

File: doc/sphinx-guides/source/installation/config.rst [hunk 1]
```diff
@@ -600,7 +600,7 @@ Reported Working S3-Compatible Storage
   possibly slow) https://play.minio.io:9000 service.
 
 `StorJ Object Store <https://www.storj.io>`_
- StorJ is a distributed object store that can be configured with an S3 gateway. Per the S3 Storage instructions above, you'll first set up the StorJ S3 store by defining the id, type, and label. After following the general installation, set the following configurations to use a StorJ object store: ``dataverse.files.<id>.payload-signing=true`` and ``dataverse.files.<id>.chunked-encoding=false``.
+ StorJ is a distributed object store that can be configured with an S3 gateway. Per the S3 Storage instructions above, you'll first set up the StorJ S3 store by defining the id, type, and label. After following the general installation, set the following configurations to use a StorJ object store: ``dataverse.files.<id>.payload-signing=true`` and ``dataverse.files.<id>.chunked-encoding=false``. See https://docs.storj.io/dcs/how-tos/dataverse-integration-guide/
```
Comment (qqmyers) [inline doc/sphinx-guides/source/installation/config.rst:603]: add " for step-by-step instructions." to make a complete sentence?

**Review by qqmyers — APPROVED** [review]
Looks good - just adding a URL with more detailed instructions (which is great!). The docs build and look OK. I suggested a complete sentence but that's pretty minor.

File: doc/sphinx-guides/source/installation/config.rst (hunk 1 above)
Comment (amozoss) [reply doc/sphinx-guides/source/installation/config.rst]: Yes, that is a good idea. Done

**Review by pdurbin — APPROVED** [review]
This preview looks good to me: https://dataverse-guide--8865.org.readthedocs.build/en/8865/installation/config.html#reported-working-s3-compatible-storage

**Pull request merged**